bigchaindb/commands/bigchaindb.py

```diff
@@ -278,6 +292,16 @@ def create_parser():
                                help='The backend to use. It can be either '
                                     'rethinkdb or mongodb.')
 
+    validator_parser = subparsers.add_parser('upsert-validator',
+                                             help='Add/update/delete a validator')
+
+    validator_parser.add_argument('public_key',
+                                  help='Public key of the new validator.')
+
+    validator_parser.add_argument('power',
+                                  type=int,
+                                  help='Voting power of the validator.')
```

Comment:
Maybe add a short second sentence about how setting it to zero will delete the validator, e.g.

`'Voting power of the validator. Setting it to 0 will delete the validator.'`